### PR TITLE
[AD-447] Add TUI buttons to remove current wallet/account

### PR DESCRIPTION
**Pull request by vcanadi**
**Description:** Add TUI buttons to remove current wallet/account

**YT issue:** https://issues.serokell.io/issue/AD-447

**Checklist:**

- [x] Updated docs if necessary
  - [x] [README](README.md)
  - [x] [TUI usage guide](docs/usage-tui.md)
  - [x] [Configuration documentation](docs/configuration.md)
- [x] My code complies with the [style guide](docs/code-style.md)
- [x] Tested my changes if they modify code

### CODE REVIEW

File: ui/vty-lib/src/Ariadne/UI/Vty/Widget/Wallet.hs [hunk 1]
```diff
@@ -396,6 +426,15 @@ performRename = do
...
 
+performRemove :: WidgetEventM WalletWidgetState p ()
+performRemove = do
+  UiLangFace{..} <- use walletLangFaceL
+  use walletRemoveResultL >>= \case
+    RemoveResultWaiting _ -> pass
+    _ -> liftIO (langPutUiCommand $ UiRemove) >>=
+      assign walletRemoveResultL . either RemoveResultError RemoveResultWaiting
+
+
```
Comment (gromakovsky) [inline ui/vty-lib/src/Ariadne/UI/Vty/Widget/Wallet.hs]: Extra empty line, remove please.

File: ui/vty-lib/src/Ariadne/UI/Vty/Widget/Account.hs [hunk 2]
```diff
@@ -283,6 +312,14 @@ performRename = do
     _ -> liftIO (langPutUiCommand $ UiRename $ UiRenameArgs name) >>=
       assign accountRenameResultL . either RenameResultError RenameResultWaiting
 
+performRemove :: WidgetEventM AccountWidgetState p ()
+performRemove = do
+  UiLangFace{..} <- use accountLangFaceL
+  use accountRemoveResultL >>= \case
+    RemoveResultWaiting _ -> pass
+    _ -> liftIO (langPutUiCommand $ UiRemove) >>=
```
Comment (maksbotan) [inline ui/vty-lib/src/Ariadne/UI/Vty/Widget/Account.hs]: Redundant `$`?

**Comment by gromakovsky** [comment]
@vcanadi btw, when you force push, please notify relevant people explicitly, because GitHub doesn't send notifications in this case.

**Comment by gromakovsky** [comment]
@vcanadi please rebase on latest `master` and sign your commits, after that you'll be able to merge this PR.

**Pull request merged**